virtual-desktop-enhancer.ahk

```diff
@@ -354,6 +358,20 @@ OpenDesktopManager() {
     Send #{Tab}
 }
 
+; Let the user change desktop names with a prompt, without having to edit the 'settings.ini'
+; file and reload the program.
+; The changes are temprorary (names will be overwritten by the default values of
+; 'settings.ini' when the program will be restarted.
+ChangeDesktopName() {
+    currentDesktopNumber := _GetCurrentDesktopNumber()
+    currentDesktopName := _GetDesktopName(currentDesktopNumber)
+    InputBox, newDesktopName, Change desktop name, Change the desktop name of desktop #%currentDesktopNumber%, , , , , , , , %currentDesktopName%
```

Comment:
Also I think you should change the title of the popup to "Windows 10 Virtual Desktop Enhancer", to represent the program properly. It's body is enough to relay the message about what the user is doing.